### ISSUE TITLE
[REQ] Having a mod marketplace for when the website is ready and everything

### ISSUE THREAD

**Issue by BlueOutside** [pizzaboxer/bloxstrap#2753]
### Acknowledgement of preliminary instructions

- [X] I have read and acknowledged the preliminary instructions.

### What idea do you have?

Title and basically the way it would work is you have some Requirements for the early days of the marketplace (ex. being a known modder on the server) and then if you met those you can publish them here
And it would be even better if there was a section in the app directly
Thats pretty much what i thought of rn :>

**Comment by pizzaboxer** [comment]
Yep, I've already been thinking about that long ago. It'll be a while before I get around to doing that though.
